### PR TITLE
feat: added routes for login and dashboard

### PR DESCRIPTION
**Pull request by adityadafe**
for neutron :)

### CODE REVIEW

**Review by adityadafe — COMMENTED** [review]
LGTM :)

File: client/src/main.tsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import App from './app.tsx'
+import App from './App.tsx'
```
Comment (SazedWorldbringer) [inline client/src/main.tsx]: App.tsx => app.tsx

File: client/src/App.tsx [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+
+const App:React.FC = () => {
   return (
-  <div className="text-3xl">Hello, world</div>
+ <Routes>
+  <Route path='/' element={<Auth/>}/>
+  <Route path='dashboard' element={<DashBoard/>} /> 
+ </Routes>
   )
 }
```
Comment (SazedWorldbringer) [inline client/src/App.tsx]: rename App.tsx to app.tsx

**Review by SazedWorldbringer — APPROVED** [review]

**Comment by neutr0ncodes** [comment]
dumb kakashi

**Pull request merged**